.github/ISSUE_TEMPLATE.md

```diff
@@ -23,3 +23,8 @@
 - [ ] liberty-12.2
 - [ ] liberty-12.1
 - [ ] kilo
+
+##### DocImpact: If your change requires documentation, check one of the following options:
```

Comment:
This line doesn't make sense if there is a checkbox for when no documentation is required.

In general this template is used by the issue creator, it might be worth adding a clarification that this section is to be completed by the issue owner as part of the issue's resolution.